pkg/scalers/gcp_cloud_tasks_scaler.go

```diff
@@ -0,0 +1,184 @@
+package scalers
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/go-logr/logr"
+	v2 "k8s.io/api/autoscaling/v2"
+	"k8s.io/metrics/pkg/apis/external_metrics"
+
+	kedautil "github.com/kedacore/keda/v2/pkg/util"
+)
+
+const (
+	cloudTasksStackDriverQueueSize = "cloudtasks.googleapis.com/queue/depth"
+
+	cloudTaskDefaultValue = 100
+)
+
+type cloudTasksScaler struct {
```

Comment:
Sorry for nit picking. 

Can we please include Google or Gcp In the name, so it is clear what is this structure about?

`GoogleTasksScaler` or `GcpTasksScaler` or something like this.

Could you please change this in the whole file and also in the test/e2e test files for all relevant properties/names? Thanks!

Reply:
Ok, will resolve all when my holidays end. Thank you!

Reply:
@jmalvarezf-lmes no worries at all, I completely understand. I was away almost 2 months 😄